canonical-web-and-design/maas.io
# Improve meta values via template inheritance

## Done
- Pass twitter and og values blocks from the page to the base template
- *Drive by*: Move head include into base template
- *Drive by*: Remove a nugatory image from the docs

## QA
- Open the demo
- Go to /docs/snap/3.1/ui/maas-installation
- Check you don't notice a difference in the search row now the image has been removed
- View source and see the twitter and og title is using the page title unlike production
- Check the twitter and og descriptions are not present as it will fallback to page content (*Which I hope once we replace the RAD table will be much better*)
- Check the social meta details remain the same on none docs pages such as how-it-works.
- Extra bonus point: try of the demo and production at https://www.opengraph.xyz/ and see its better for docs.

## Issue / Card
Fixes https://github.com/canonical-web-and-design/maas.io/issues/644
